tools.md

```
@@ -0,0 +1,10 @@
+# Ferramentas
+
+## Coderabbit
+- https://docs.coderabbit.ai/getting-started/configure-coderabbit/
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)